modules/cluster-logging-uninstall.adoc

```diff
@@ -15,17 +15,96 @@ You can remove cluster logging from your cluster.
 
 To remove cluster logging:
 
-. Use the following command to remove everything generated during the deployment.
+. Use the following command to remove the Cluster Logging CR.
 +
 [source,terminal]
 ----
 $ oc delete clusterlogging instance -n openshift-logging
 ----
 
-. Use the following command to remove the Persistent Volume Claims that remain
-after the Operator instances are deleted:
+. Optional: Use the following command to remove any Cluster Logging Persistent Volume Claims:
```

Comment:
Delete only pvc for clusterlogging
oc delete pvc --selector=logging-cluster=elasticsearch -n openshift-logging